scylladb/care-pet
# Add sphinx theme 1.0

Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/133

Sphinx ScyllaDB Theme 1.0 is now released 🥳 

We’ve made a number of updates to the look and feel of the theme to improve the overall user experience.
You can read more about all notable changes [here](https://sphinx-theme.scylladb.com/stable/CHANGELOG#september-2021).

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser. You will see the docs with the new look and feel:

![image](https://user-images.githubusercontent.com/9107969/134825940-88a64486-c5dd-4470-be1c-482b64ff5780.png)

## Before you merge this PR...

I've proposed a new landing page for the project (index.rst). Please, review it and let me know if you would like to change the distribution of the cards, change the text, or use different icons/images.
